test/models.py

```diff
@@ -95,6 +95,10 @@ class AggregateTestModel(Model):
 
 
 class AggregateTestChildModel(Model):
-    parent = ForeignKey('AggregateTestModel', related_name='children')
+    parent = ForeignKey(
+        'AggregateTestModel',
+        related_name='children',
+        on_delete=CASCADE,
```

Comment:
What's the purpose of this?

Reply:
This was cargo culted from #74 to be honest, but the root cause is because of a breaking change in Django 2.0 where `ForeignKey` now requires on_delete kwarg.

> The on_delete argument for ForeignKey and OneToOneField is now required in models and migrations.

Reply:
OK. I forgot about that